src/runtime/parallel/coordinator.ts

```diff
@@ -94,6 +111,26 @@ export default class Coordinator implements IRuntime {
     }
   }
 
+  awakenWorkers(triggeringWorker: IWorker): void {
+    Object.values(this.workers).forEach((worker) => {
+      if (worker.state === WorkerState.idle) {
+        this.giveWork(worker)
+      }
+      return worker.state !== WorkerState.idle
+    })
+
+    let wip: Boolean = false
```

Comment:
Could these few lines not effectively be replaced with:

```ts
const wip = this.inProgressPickles.length > 0
```

Or am I missing something?

Reply:
`inProgressPickles` is an object. Having to replace the lodash implementation, I found this is the most performant solution.